### DISCUSSION TITLE
su'aal

### DISCUSSION THREAD

**Discussion by mucaadciiltire** [sharafdin/markdown-master#7] [Q&A] [answered]
wll maxaa iga caawin kara in aan dhisto mid aan aniga leeyahay

**Comment by omartood** [comment, marked as answer]
wll waxaad u baahantay inaad baratid javascript ama typescript bas.

**Reply by mucaadciiltire** [reply]
haye wll. in aad i caawiso rajeynaa mhdsnid

**Reply by omartood** [reply]
Insha ' allah
